.rubocop.yml

```diff
@@ -7,3 +7,6 @@ AllCops:
 
 ParameterLists:
   CountKeywordArgs: false
+
+Metrics/ModuleLength:
```

Comment:
What are the steps to inherit from the up-to-date CocoaPods config as mentioned by segiddins?